x-pack/plugins/cases/public/components/case_action_bar/status_context_menu.tsx

```diff
@@ -55,19 +57,24 @@ const StatusContextMenuComponent: React.FC<Props> = ({
     [closePopover, currentStatus, onStatusChanged]
   );
 
+  // TODO: Determine if we would prefer to show the disabled options with grayed out treatment
   const panelItems = useMemo(
     () =>
-      caseStatuses.map((status: CaseStatuses) => (
-        <EuiContextMenuItem
-          data-test-subj={`case-view-status-dropdown-${status}`}
-          icon={status === currentStatus ? 'check' : 'empty'}
-          key={status}
-          onClick={() => onContextMenuItemClick(status)}
-        >
-          <Status status={status} />
-        </EuiContextMenuItem>
-      )),
-    [currentStatus, onContextMenuItemClick]
+      caseStatuses
+        .filter(
+          (status: CaseStatuses) => !shouldDisableStatusFn([{ status: currentStatus }], status)
```

Comment:
@cnasikas - For now, chose to just not show the actions as not sure what we would want the disabled state to look like for the popover elements. Happy to make any changes based on your feedback here :)